pkgs/development/compilers/rust/binary.nix

```diff
@@ -71,6 +72,7 @@ rec {
 
     meta = with lib; {
       homepage = "http://www.rust-lang.org/";
+      sourceProvenance = with sourceTypes; [ binaryNativeCode ];
       description = "A safe, concurrent, practical language";
```

Comment:
This description is also obviously wrong.

Reply:
How so?

Reply:
This is cargo, not rust.

Reply:
fixed.

Reply:
Thanks! :)